contracts/plugins/governance/Governance.sol

```diff
@@ -141,19 +149,13 @@ contract Governance is
         return super._executor();
     }
 
-    /// @return {micro %} The portion of the StRSR supply the account had at a previous blocknumber
+    /// @return {qStRSR} The voting weight the account had at a previous block number
```

Comment:
Why did we move away from micropercents? Wasnt that on purpose that we implemented it in that way?

```
* Note that due to the elastic supply of StRSR, proposalThreshold is handled
 *   very differently than the typical approach. It is in terms of micro %,
 *   as is _getVotes().
 ```